src/main/java/org/openelisglobal/audittrail/controller/AuditTrailReportRestController.java

```diff
@@ -0,0 +1,38 @@
+package org.openelisglobal.audittrail.controller;
+ 
+import java.util.List;
+
+import java.util.HashMap;
+ 
+import java.util.Map;
+
+import org.openelisglobal.audittrail.action.workers.AuditTrailItem;
+import org.openelisglobal.audittrail.action.workers.AuditTrailViewWorker;
+ 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+
+@RestController
+public class AuditTrailReportRestController {
+
+    @GetMapping("/rest/AuditTrailReport")
+    public ResponseEntity<Map<String, Object>> getAuditTrailReport(@RequestParam String accessionNumber) {
+        Map<String, Object> response = new HashMap<>();
```

Comment:
Is there any reason why you ddnt use the AuditTrailViewForm object ??
We want to avoid loosely typed scenarios.

also put this REST controller under a subfolder called rest to be consisted with other REST controllers